warehouse/models/payments_views_staging/stg_cleaned_authorisations.sql

```diff
@@ -0,0 +1,11 @@
+WITH stg_cleaned_authorisations AS (
+    SELECT DISTINCT * EXCEPT (
```

Comment:
I don't love `select distinct *` models; is there a primary key we can use to de-dupe instead?

Reply:
there is not, aside from `calitp_hash`, which I believe we're trying to get away from? would it be worthwhile to use it here?

Reply:
ah no that's fine then, we can leave for now